packages/react/src/__tests__/ReactProfiler-test.internal.js

```diff
@@ -3923,7 +3923,6 @@ describe('Profiler', () => {
 
         expect(onPostCommit).toHaveBeenCalledTimes(1);
         expect(onPostCommit.mock.calls[0][4]).toMatchInteractions([
-          initialRenderInteraction,
```

Comment:
This is just gone?

Reply:
tl;dr the initial render is no longer entangled with the Suspense retry. See rationale from previous comment.